openedx/core/djangoapps/discussions/models.py

```diff
@@ -418,3 +417,55 @@ def available_providers(self) -> list[str]:
     @classmethod
     def get_available_providers(cls, context_key: CourseKey) -> list[str]:
         return ProviderFilter.current(course_key=context_key).available_providers
+
+
+class ProgramDiscussionsConfiguration(TimeStampedModel):
+    """
+    Associates a program with a discussion provider and configuration
+    """
+
+    program_uuid = models.CharField(
+        primary_key=True,
+        db_index=True,
+        unique=True,
+        max_length=255,
```

Comment:
are you sure about the max_length?

Reply:
Just using the standard charField length here.

Reply:
The length of the data is known, so, you shouldn't set it to 255 which can never happen. For known lengths its better to use those rather than than standard.